services/movie_session.py

```diff
@@ -42,3 +42,11 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(
+        movie_session_id: int
+) -> list[dict]:
```

Comment:
The return type annotation `list[dict]` is correct, but ensure that the dictionaries contain the expected keys ('seat', 'row').